modules/core/src/main/java/org/apache/ignite/internal/commandline/tracing/configuration/TracingConfigurationArguments.java

```diff
@@ -0,0 +1,178 @@
+        return cmd;
+    }
+
+    /**
+     * Builder of {@link TracingConfigurationArguments}.
+     */
+    @SuppressWarnings("PublicInnerClass") public static class Builder {
+        /** Counterpart of {@code TracingConfigurationArguments}'s command. */
+        private TracingConfigurationSubcommand cmd;
+
+        /** Counterpart of {@code TracingConfigurationArguments}'s scope. */
+        private Scope scope;
+
+        /** Counterpart of {@code TracingConfigurationArguments}'s label. */
+        private String lb;
+
+        /** Counterpart of {@code TracingConfigurationArguments}'s samplingRate. */
+        private double samplingRate;
+
+        /** Counterpart of {@code TracingConfigurationArguments}'s supportedScopes. */
+        private Set<Scope> supportedScopes;
+
+        /**
+         * Constructor.
+         *
+         * @param cmd {@code TracingConfigurationSubcommand} command.
+         */
+        public Builder(TracingConfigurationSubcommand cmd) {
+            this.cmd = cmd;
+        }
+
+        /**
+         * Builder method that allows to set scope.
+         * @param scope {@link Scope} of a trace's root span
+         *  to which some specific tracing configuration will be applied.
+         * @return Builder.
+         */
+        public @NotNull Builder withScope(Scope scope) {
+            this.scope = scope;
+
+            return this;
+        }
+
+        /**
+         * Builder method that allows to set sampling rate.
+         *
+         * @param samplingRate Number between 0 and 1 that more or less reflects the probability
+         *  of sampling specific trace.
+         *  0 and 1 have special meaning here, 0 means never 1 means always. Default value is 0 (never).
+         * @return Builder.
+         */
+        public @NotNull Builder withSamplingRate(double samplingRate) {
+            this.samplingRate = samplingRate;
+
+            return this;
+        }
+
+        /**
+         * Builder method that allows to set optional label attribute.
+         *
+         * @param lb Label of traced operation. It's an optional attribute.
+         * @return Current {@code TracingConfigurationCoordinates} instance.
+         */
+        public @NotNull Builder withLabel(@Nullable String lb) {
+            this.lb = lb;
+
+            return this;
+        }
+
+        /**
+         * Builder method that allows to set supported scopes.
+         *
+         * @param supportedScopes Set of {@link Scope} that defines which sub-traces will be included in given trace.
+         * In other words, if child's span scope is equals to parent's scope
+         * or it belongs to the parent's span supported scopes, then given child span will be attached to the current trace,
+         * otherwise it'll be skipped.
+         * See {@link Span#isChainable(org.apache.ignite.internal.processors.tracing.Scope)} for more details.
+         * @return {@code TracingConfigurationParameters} instance.
```

Comment:
This method returns a builder itself, please correct JavaDoc here and below

Reply:
Fixed.